.github/pull_request_template.md

```diff
@@ -1,62 +1,63 @@
-> Hi there! Thanks for submitting a pull request to Concourse!
+<!---
+Hi there! Thanks for submitting a pull request to Concourse!
+To help us review your PR, please fill in the following information.
+-->
+
+## What does this PR accomplish?
+<!---
+Choose all that apply.
+Also, mention the linked issue here.
+This will magically close the issue once the PR is merged.
+-->
+Bug Fix | Feature | Documentation
 
-> If you haven't already, please take a look at our [Code of Conduct] and our
-> [Contributing Guide]. There are certain anti-patterns that we try to avoid -
-> check the [Anti-patterns page] to know more.
-
-> To help us review your PR, please fill in the following information.
-
-> Also, before submitting, please remove all the lines starting with '>'.
+closes # .
 
+## Changes proposed by this PR:
+<!---
+Tell the reviewer What changed, Why, and How were you able to accomplish that?
+-->
+- ...
```

Comment:
I'd rather keep this space empty so the contributor can start writing right away without having to remove the dots, and the hyphen if they're not using bullet points.

Reply:
By "space" I mean line 20 - the "-..."

Reply:
done